code/modules/hydroponics/fermenting_barrel.dm

```diff
@@ -4,7 +4,7 @@
 	icon = 'icons/obj/objects.dmi'
 	icon_state = "barrel"
 	density = TRUE
-	anchored = FALSE
+	anchored = TRUE
```

Comment:
Undocumented change in the PR description or CL.
Do please add this in the CL and description that barrels now spawn anchored.

Reply:
Forgot about that.